layouts/partials/footer.html

```diff
@@ -28,6 +28,18 @@ <h3>Email Address</h3>
     </div>
   </div>
 </div>
+
```

Comment:
Need to add this, otherwise, the featured footer will be overridden by the footer.html